src/pages/home/components/Sidebar/components/SocialMediaForm/SocialMediaForm.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import { Story } from '@ladle/react';
+
+import SocialMediaForm from './SocialMediaForm';
+
+export const SocialMediaFormStories: Story = () => (
+  <div style={{ width: 680 }}>
+    <SocialMediaForm
+      onHandleToggleModal={function (): void {
+        throw new Error('Function not implemented.');
+      }}
```

Comment:
Não consegue deixar assim?
```suggestion
    <SocialMediaForm
      onHandleToggleModal={(): void => throw new Error('Function not implemented.')}
```

Reply:
refatorado para arrow function